analysis_driver/quality_control/calculate_relatedness.py

```diff
@@ -1,41 +1,50 @@
-from egcg_core import executor, util
+import os
+from egcg_core import executor, util, clarity
 from luigi import Parameter, ListParameter
 from analysis_driver.config import default as cfg
 from analysis_driver.segmentation import Stage
 from analysis_driver.util.bash_commands import java_command
 
-
-class Relatedness(Stage):
-    gvcf_files = ListParameter()
+class Genotype_gVCFs(Stage):
+    gVCFs = ListParameter()
     reference = Parameter()
-    project_id = Parameter()
 
     @property
     def gatk_outfile(self):
```

Comment:
`gatk_outfile` should be put to a parent class that `Peddy`, `Relatedness`, and `Genotype_gVCFs` would subclass.